### ISSUE TITLE
Spyder incompatibility with FEniCS+scikit-image when installation uses conda

### ISSUE THREAD

**Issue by ozanoktem** [spyder-ide/spyder#4374]
## Description of your problem
There seems to be an incompatibility that appears when installing FEniCS + spyder + scikit-image (test 2 below) using conda on Ubuntu 16.04 LTS. The symptom is that spyder does not launch, see proper bug report below. 

**What steps will reproduce the problem?**

**Test 1:** Install FEniCS + spyder using conda. Details outlined below.
_Outcome:_ FEniCS and spyder seem to work just fine. 

1. Create environment with spyder
```
> conda create -n fenics python=3.5 spyder
> source activate fenics
```

2. Test spyder
`> spyder
`
_Result:_ Launches without warnings.

3. Install FEniCS using conda
`> conda install -c conda-forge fenics=2016.2.0 
`
_Result:_ Installation succesful, but the following warning messages were issued:

```
The following packages will be UPDATED:
    icu:         54.1-0                                    --> 58.1-1        conda-forge

The following packages will be SUPERCEDED by a higher-priority channel:
    pyqt:        5.6.0-py35_2                              --> 4.11.4-py35_2 conda-forge
    qtconsole:   4.3.0-py35_0                              --> 4.3.0-py35_0  conda-forge

The following packages will be DOWNGRADED due to dependency conflicts:
    qt:          5.6.2-3                                   --> 4.8.7-3  
```

4. Test spyder
`> spyder
`
_Result:_ Launches with the following warning:

`GLib-GIO-Message: Using the 'memory' GSettings backend.  Your settings will not be saved or shared with other applications.
`

5. Update conda
`> conda update --all
`
_Result:_ Sucess

6. Test spyder
`> spyder
`
_Result:_ Launches with the same warning as in step (4) above.

**Test 2:** Install FEniCS + spyder + scikit-image using conda. Details outlined below.
_Outcome:_ Spyder does not launch.

1. Create environment with spyder and scikit-image
```
> conda create -n fenics-scikit-image python=3.5 spyder scikit-image
> source activate fenics-scikit-image
```
2. Test spyder
`> spyder
`
_Result:_ Launches without warnings.

3. Install FEniCS using conda
`> conda install -c conda-forge fenics=2016.2.0 
`
_Result:_ Installation succesful, but the following warning messages were issued:

```
The following packages will be UPDATED:
    fontconfig:  2.12.1-3                            --> 2.12.1-4                           conda-forge
    freetype:    2.5.5-2                             --> 2.7-1                              conda-forge
    icu:         54.1-0                              --> 58.1-1                             conda-forge
    libpng:      1.6.27-0                            --> 1.6.28-0                           conda-forge
    matplotlib:  2.0.0-np112py35_0                   --> 2.0.0-np112py35_3                  conda-forge
    numpy:       1.12.1-py35_0                       --> 1.12.1-py35_blas_openblas_200      conda-forge [blas_openblas]
    scipy:       0.19.0-np112py35_0                  --> 0.19.0-np112py35_blas_openblas_200 conda-forge [blas_openblas]

The following packages will be SUPERCEDED by a higher-priority channel:
    pillow:      4.1.0-py35_0                        --> 4.1.0-py35_0                       conda-forge
    qt:          5.6.2-3                             --> 5.6.2-1                            conda-forge
```
4. Test spyder
`> spyder
`
_Result:_ Fails with the following message:

`Segmentation fault (core dumped)
`
5. Update conda
`> conda update --all
`
_Result:_ Sucess

6. Test spyder
`> spyder
`
_Result:_ Fails with the same message as in step (4) above.

**What is the expected output? What do you see instead?**
I expected spyder to launch, instead it core dumps.

**Please provide any additional information below**

## Versions and main components

* Spyder Version: 3.1.3
* Python Version: 3.5
* Qt Version: 5.6.2
* PyQt Version: 5.6.0
* Operating system: Ubuntu 16.04 LTS

## Dependencies

Please go to the menu entry `Help > Optional Dependencies` (or
`Help > Dependencies`), press the button `Copy to clipboard`
and paste the contents below:

**Comment by ccordoba12** [comment]
This is caused by an incompatibility between conda-forge and defaults. You don't really need to install Spyder in the environment where you're trying to install Fenics.

You simply need to change the Python interpreter used by Spyder to create Python and IPython consoles in

    Tools > Preferences > Python interpreter

**Comment by hsllo** [comment]
I changed the interpreter as: Tools > Preferences > Python interpreter to the fenicsproject environment but is not working with Spyder. However, the same process when using Pycharm works. Any clue why it does not work with Spyder?

**Comment by ccordoba12** [comment]
Hi,

This problem will be fixed in our next version (4.0.2), to be released 
in a couple of weeks.


Cheers,
Carlos
